sunpy/timeseries/timeseriesbase.py

```diff
@@ -68,25 +68,23 @@ class GenericTimeSeries:
     """
     A generic time series object.
 
+    Parameters
+    ----------
+    data : `~pandas.DataFrame`
+        A pandas DataFrame representing one or more fields as a function
+        of time.
+    meta : `~sunpy.timeseries.metadata.TimeSeriesMetaData`, optional
+        The metadata giving details about the time series data/instrument.
+    **kwargs : dict
+        Keyword arguments are not used.
```

Comment:
You can remove this from here, they don't need documenting if they don't do anything.

IIRC, the reason `**kwargs` is here is because sometimes the factory passes extra things through, which maybe it shouldn't but is not a trivial fix.